versioned_sidebars/version-3.9-sidebars.json

```
@@ -141,40 +289,132 @@
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalardl-schema-loader",
+                  "label": "Configure a Custom Values File for ScalarDL Schema Loader"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-admin-for-kubernetes",
+                  "label": "Configure a Custom Values File for Scalar Admin for Kubernetes"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-scalar-manager",
+                  "label": "Configure a Custom Values File for Scalar Manager"
+                },
+                {
+                  "type": "doc",
+                  "id": "helm-charts/configure-custom-values-envoy",
+                  "label": "Configure a Custom Values File for Scalar Envoy"
+                }
               ]
             },
-            "helm-charts/how-to-deploy-scalar-products",
-            "helm-charts/how-to-deploy-scalardl-ledger",
-            "helm-charts/how-to-deploy-scalardl-auditor",
-            "helm-charts/how-to-deploy-scalar-admin-for-kubernetes",
-            "helm-charts/mount-files-or-volumes-on-scalar-pods",
-            "helm-charts/use-secret-for-credentials"
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-products",
+              "label": "Deploy Scalar Products Using Scalar Helm Charts"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-ledger",
+              "label": "Deploy ScalarDL Ledger"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalardl-auditor",
+              "label": "Deploy ScalarDL Auditor"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/how-to-deploy-scalar-admin-for-kubernetes",
+              "label": "Deploy Scalar Admin for Kubernetes"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/mount-files-or-volumes-on-scalar-pods",
+              "label": "Mount Files or Volumes on ScalarDL Pods"
+            },
+            {
+              "type": "doc",
+              "id": "helm-charts/use-secret-for-credentials",
+              "label": "Use Secret Resources"
+            }
           ]
         },
-        "scalar-licensing/README",
-        "installation-with-docker",
-        "ca/caserver-getting-started"
+        {
+          "type": "doc",
+          "id": "scalar-licensing/README",
+          "label": "Configure a Product License Key"
+        },
+        {
+          "type": "doc",
+          "id": "installation-with-docker",
+          "label": "Install ScalarDL in Your Local Environment"
```

```suggestion
          "label": "Install ScalarDL in Your Local Environment with Docker"
```
Same as `sidebars.js`.